recipes/libvips/README.md

```diff
@@ -0,0 +1,160 @@
+<!--
+# -*- mode: jinja -*-
+-->
+
+About libvips
```

Comment:
I think this is going to get clobbered. Consider making it more succinct and adding it to the recipe.